src/item.cpp

```diff
@@ -2549,25 +2695,48 @@ void item::component_info( std::vector<iteminfo> &info, const iteminfo_query *pa
     }
 }
 
+void item::repair_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
+                        int /*batch*/, bool /*debug*/ ) const
+{
+    if( !parts->test( iteminfo_parts::DESCRIPTION_REPAIREDWITH ) ) {
+        return;
+    }
+    insert_separation_line( info );
+    const std::set<std::string> &rep = repaired_with();
+    if( !rep.empty() ) {
+        info.emplace_back( "DESCRIPTION", _( "Repair using " ) +
+        enumerate_as_string( rep.begin(), rep.end(), []( const itype_id & e ) {
+            return nname( e );
+        }, enumeration_conjunction::or_ ) + "." );
```

Comment:
```
        info.emplace_back( "DESCRIPTION", string_format( _( "Repair using %s." ),
        enumerate_as_string( rep.begin(), rep.end(), []( const itype_id & e ) {
            return nname( e );
        }, enumeration_conjunction::or_ ) ) );
```
So it can be properly translated in other languages

Reply:
Yeah I wondered if there were a better way to do this. Will include.

Reply:
Added in https://github.com/CleverRaven/Cataclysm-DDA/pull/38030/commits/daf2a84b485ca1b87aa4faef0b75092aecd2dba8